CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ function(build_ngraph)
     if (UNIX)
         ie_add_compiler_flags(-Wno-error=return-type -Wno-undef)
     elseif(WIN32)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4308 /wd4146 /wd4703 /wd4244")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4308 /wd4146 /wd4703 /wd4244 /wd4819")
```

Comment:
may we need to add this suppression only for 32bits generators?

Reply:
WIN32 is  defined as 1 when the compilation target is x86 or x64 windows programs, not just the 32bits